src/components/m-dropdown/mixins/dropdown.scss

```diff
@@ -145,6 +145,11 @@
     position: relative;
     right: auto;
   }
+  
+  @include target(moz) {
```

Comment:
@LucaMele 
@jan-capiak 
This should fix the doubled margin issue at firefox.

We still need a positive test, that layouts don't blow up on mobiles